### ISSUE TITLE
USWDS - Usability Testing: Links styled as buttons findings

### ISSUE THREAD

**Issue by mahoneycm** [uswds/uswds#5527]
## Summary
A link styled as a button was erroneously used for "Next step" in the form prototype when we conducted usability testing with visually impaired participants (see [findings report](https://github.com/uswds/uswds/wiki/Usability-research-findings-for-Q3-2023)), and participants struggled with finding and interacting with it due to it being coded as a link. We will update the prototype and test again with users, and also conduct manual accessibility testing to see how different screen readers handle links with `role="button"`. VoiceOver doesn't have a quick jump-to-buttons command. 

## Observations

All participants who used a screen reader (4) had problems with finding or interacting with the sign-in link.

- People expect the ‘submit’ or ‘sign in’ to be an actual button, so that is what they look for.
- Often, screen reader users use keyboard shortcuts to navigate certain elements (like “b” to find all the buttons). If it is coded as a link, it doesn’t take them there.
- One person made the point that sometimes there are sites that have hundreds of links (ex: CNN), so if the sign in is actually a link, it’d be easy to lose it or take forever to get to it. 
    
## Note
WCAG recommends that all links styled as buttons should have the button role for accessibility purposes
>    The types of actions performed by buttons are distinctly different from the function of a link (see [link pattern](https://www.w3.org/WAI/ARIA/apg/patterns/link/). It is important that both the appearance and role of a widget match the function it provides. Nevertheless, elements occasionally have the visual style of a link but perform the action of a button. In such cases, giving the element role `button` helps assistive technology users understand the function of the element. However, a better solution is to adjust the visual design so it matches the function and ARIA role. [^1]
    
[^1]: https://www.w3.org/WAI/ARIA/apg/patterns/button/

If this is acceptable, we'll need to update guidance around links styled as buttons. Also, the test forms did not have the button role.

## Next steps
- [x] [Zombie prototype form - Update next step buttons](https://github.com/uswds/uswds-sandbox/issues/113) - once updated we will test again in the next round of usability testing
- [ ] Conduct manual testing to see how different screen readers handle links with `role="button"`. VoiceOver doesn't have a quick jump-to-buttons command.
- [ ] Update guidance to say that all links styled as buttons should have `role="button"`, depending on results of manual testing.

**Comment by ryparker** [comment]
The difficulties participants had finding the form action are still worth following up on. The prototype has been corrected, but we still need the retest results and any resulting guidance changes.

Let's check whether the corrected action and current examples are understandable. Navigation should use links, and actions such as submitting a form should use buttons. Styling a link to look like a button isn't, by itself, a reason to add `role=button`.
